### ISSUE TITLE
Speckle support

### ISSUE THREAD

**Issue by lettore** [yorikvanhavre/WebTools#18]
Hello, I see there's a wip Speckle connector in this repo, as I'm using Speckle for share data between Rhinoceros and other CAD programs I will love to see Speckle supported in Freecad.

**Comment by yorikvanhavre** [comment]
Yes, that work got stopped at some point because the Speckle API kept developing faster than I could code and I couldn't keep the pace :sweat_smile: but definitely, now it's pretty much stabilized, and we should restart working on this

**Comment by lettore** [comment]
That will be amazing, there's not much I can help but if you need a beta tester I'm on the boat
